README.md

```diff
@@ -24,27 +24,15 @@ This work will be presented at [NeurIPS 2024](https://neurips.cc/virtual/2024/po
 **Note: This repository will be updated further during the next week to make it easier to set up and use the four different databases.**
 
 ## Data
+The `./data` directory contains various used templates, synthea data, train and dev data and the compoted results.
```

Comment:
Fix to:
The `./data` directory contains our template questions, the Synthea data, train and dev data, and the Text-to-Query results of our evaluated LLMs for all databases.